contracts/scripts/deployments/diamonds/DeployRiverRegistry.s.sol

```diff
@@ -132,6 +141,35 @@ contract DeployRiverRegistry is DiamondHelper, Deployer {
       });
   }
 
+  function diamondInitParamsFromFacets(
+    address deployer,
+    string[] memory facets
+  ) public {
+    for (uint256 i = 0; i < facets.length; i++) {
+      string memory facetName = facets[i];
+      address facetHelperAddress = facetDeployments[facetName];
+      if (facetHelperAddress != address(0)) {
+        // deploy facet
+        address facetAddress = Deployer(facetHelperAddress).deploy(deployer);
```

Comment:
The Facet helper derives both from Deployer, FacetHelper but I need to call `deploy` here. Not sure if there's a better way to infer the interface than this.